src/content/docs/cypher/expressions/predicate-functions.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I cannot find this page here https://dev-docs.kuzudb.com/cypher/expressions/. I think it's missing from the nav.

Also, why are these expressions instead of functions? Shouldn't they be predicate function or part of list functions?